aws-datastore/src/androidTest/java/com/amplifyframework/datastore/storage/SynchronousStorageAdapter.java

```diff
@@ -91,10 +98,22 @@ public static SynchronousStorageAdapter create(
      */
     @SuppressWarnings("UnusedReturnValue")
     public List<ModelSchema> initialize(@NonNull Context context) throws DataStoreException {
+        Log.d("initialize", "inside initialize");
         return Await.result(
             operationTimeoutMs,
             (Consumer<List<ModelSchema>> onResult, Consumer<DataStoreException> onError) ->
-                asyncDelegate.initialize(context, onResult, onError)
+            {
+                try {
+                    asyncDelegate.initialize(context, onResult, onError,
+                            DataStoreConfiguration.builder()
+                                    .syncInterval(2L,TimeUnit.MINUTES)
+                                    .observeQueryMaxRecords(2)
+                                    .observeQueryMaxTime(0)
+                                    .build());
+                } catch (DataStoreException exception) {
+                    exception.printStackTrace();
```

Comment:
Don't just print it, call. `onError.accept(exception)` (The callback hook provided by `Await.result(...)`

Reply:
Good suggestion. Thank you!